app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadViewModel.kt

```diff
@@ -101,7 +103,9 @@ class ThreadViewModel @Inject constructor(
     ).map { it.obj }.asLiveData(ioCoroutineContext)
 
     fun resetMessagesCache() {
-        cachedSplitBodies = mutableMapOf()
+        treatedMessagesForCalendarEvent.clear()
+        isCalendarEventExpandedMap.clear()
```

Comment:
We might need to rename this method because it's not obvious that the calendar is reset when we reset a message. Maybe `resetMessagesRelatedCache`?